src/LibraryManager.Vsix/UI/Controls/TargetLocation.xaml.cs

```diff
@@ -67,13 +98,6 @@ public string Text
             {
                 _text = (string)GetValue(TextProperty);
 
-                // Pre populate textBox with folder name
-                if (_text == null)
-                {
-                    _text = InstallationFolder.DestinationFolder;
-                    TargetLocationSearchTextBox.Text = _text;
-                }
-                
                 InstallationFolder.DestinationFolder = _text;
```

Comment:
>InstallationFolder.DestinationFolder = _text; [](start = 16, length = 45)

this seems weird. Can you explain a bit?

Reply:
Everything in here will go away with Yang's changes:
[https://github.com/aspnet/LibraryManager/pull/280](https://github.com/aspnet/LibraryManager/pull/280)

I'll leave this as is for now as I need it for testing.

---
In reply to: [200472998](https://github.com/aspnet/LibraryManager/pull/278#discussion_r200472998) [](ancestors = 200472998)